### DISCUSSION TITLE
Homebrew formula with .tar.gz URL from gitea repository -> no bottle found error

### DISCUSSION THREAD

**Discussion by FiskFan1999** [Homebrew/discussions#3484] [Writing Formulae/Casks]
Hey everybody, 

I created two homebrew formulas for golang projects using the `brew --go create https://example.net/.../v0.0.0.tar.gz` syntax. However, one of them currently installs and compiles correctly, and the other gets me a `Error: <name>: no bottle available!` error. I should note that the first formula is from a github repository and the second is from a gitea repository.
```
https://github.com/user/project/archive/refs/tags/v0.0.0.tar.gz
https://codeberg.org/user/project/archive/v0.0.0.tar.gz
```
This difference is puzzling to me because both applications are identical as far as I can tell, in that they would be installed via `go install`.

Is there a special case for a URL from github that will tip off brew to install from source, that doesn't get detected for the gitea repository (maybe a grep for `/refs/tags/`?), and is there a way to enable this logic for the second project?

Thanks,

**Comment by carlocab** [comment]
I don't think it has to do with the source tarball domain.

What's the output of
```
brew info github-formula
brew info gitea-formula
```
?
